src/Sylius/Behat/Context/Ui/Admin/ManagingProductVariantsContext.php

```diff
@@ -206,6 +223,35 @@ public function theProductVariantShouldAppearInTheShop($productVariantCode, Prod
     }
 
     /**
+     * @Then the :productVariantCode variant of the :product product should not appear in the shop
```

Comment:
I would prefer to keep consistency with previous [step](https://github.com/Sylius/Sylius/pull/6768/files#diff-9caa6c41a42784d0658dcbb0dbc447eaR213)